packages/react-dom-bindings/src/shared/ReactDOMFlightClientHostConfig.js

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
What will this file be used for?

The naming convention seems like it should be `ReactFlightClientHostConfigDOM` if you look at its siblings:

`ReactFlightClientHostConfigBrowser` (runtime)
`ReactFlightClientHostConfigStream` (channel)
`ReactFlightClientHostConfigDOM` (format)

The difference between Browser and DOM here is subtle though.

Should these be split between the Flight client running in Fizz and client-side?

Reply:
In my head the `*Browser` and `*Node` files were still describing the server runtime and would be appropriate to use for Native and other hosts.

`ReactFlight` is to `ReactFiber`
`ReactDOMFlight` is to `ReactDOM` in my naming scheme

I could use the naming scheme you suggest but it would still be appropriate to keep the file in `react-dom-bindings` project. And since it is not a sibling file of the other two I figured it didn't need to follow the same structure